models/glif_psc.h

```diff
@@ -68,33 +68,41 @@ GLIF model mechanism setting is based on three parameters
++--------+---------------------------+----------------------+--------------------+
+| Model  | spike_dependent_threshold | after_spike_currents | adapting_threshold |
++========+===========================+======================+====================+
+| GLIF1  | False                     | False                | False              |
++--------+---------------------------+----------------------+--------------------+
+| GLIF2  | True                      | False                | False              |
++--------+---------------------------+----------------------+--------------------+
+| GLIF3  | False                     | True                 | False              |
++--------+---------------------------+----------------------+--------------------+
+| GLIF4  | True                      | True                 | False              |
++--------+---------------------------+----------------------+--------------------+
+| GLIF5  | True                      | True                 | True               |
++--------+---------------------------+----------------------+--------------------+
 
 Typical parameter setting of different levels of GLIF models for different cells
 can be found and downloaded in the `Allen Cell Type Database
-<https://celltypes.brain-map.org>`_. For example, the default parameter setting of this
-glif_cond neuron model was from the parameter values of GLIF Model 5 of Cell
+<https://celltypes.brain-map.org>`_. For example, the default parameter setting of
+this ``glif_psc`` neuron model was from the parameter values of GLIF Model 5 of Cell
 490626718, which can be retrieved from the `Allen Brain Atlas
 <https://celltypes.brain-map.org/mouse/experiment/electrophysiology/
 490626718>`_, with units being converted from SI units (i.e., V, S (1/Ohm),
 F, s, A) to NEST used units (i.e., mV, nS (1/GOhm), pF, ms, pA) and values
 being rounded to appropriate digits for simplification.
 
 For models with spike dependent threshold (i.e., GLIF2, GLIF4 and GLIF5),
-parameter setting of voltage_reset_fraction and voltage_reset_add may lead to the
-situation that voltage is bigger than threshold after reset. In this case, the neuron
-will continue to spike until the end of the simulation regardless the stimulated inputs.
-We recommend the setting of the parameters of these three models to follow the
-condition of :math:`(E_L + voltage_reset_fraction * ( V_th - E_L ) + voltage_reset_add)
-< (V_th + th_spike_add)`.
+parameter setting of ``voltage_reset_fraction`` and ``voltage_reset_add`` may lead
+to the situation that voltage is bigger than threshold after reset. In this case,
+the neuron will continue to spike until the end of the simulation regardless the
+stimulated inputs. We recommend the setting of the parameters of these three models
+to follow the condition of
+
+.. math::
+
+    E_L + \mathrm{voltage_reset_fraction} \cdot \left( V_\mathrm{th} - E_L \right)
+    + \mathrm{voltage_reset_add} < V_\mathrm{th} + \mathrm{th_spike_add}
```

Comment:
```suggestion
    + \mathrm{voltage\_reset\_add} < V_\mathrm{th} + \mathrm{th\_spike\_add}
```
The equation does not render (error: Double subscripts: use braces to clarify). Adding the backslashes seems to do the trick. Can you also modify this equation in `glif_cond`.